### PR TITLE
Finish README

### PR DESCRIPTION
**Pull request by ghortat**
Ask question about dancing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 This is where i edit this document to try out github
 
 This is the 2nd version of the file
+
+Hortatos here! I was wondering, have you ever danced with the devil in the pale moonlight?
```
Comment (anastop) [inline README.md:6]: Have you?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 This is where i edit this document to try out github
```
Comment (anastop) [inline README.md]: I propose the following change to this file: 

```suggestion
This is where I edit this document to try out github
```

**Review by anastop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghortat) [reply README.md:6]: No, Jack Nicholson playing Joker in the 1989 film Batman did!

**Review by anastop — APPROVED** [review]
OK, your PR looks good to me! I will approve it, so go on and: 
- merge it to master (as long as there are no conflicts) 
- delete its associated branch

**Comment by ghortat** [comment]
Done! Thank you @anastop for your contribution

**Pull request merged**